atoms/Cargo.toml

```
@@ -1,10 +1,12 @@
 [package]
-name = "servo_atoms"
-version = "0.0.1"
+name = "stylo_atoms"
+version = "0.1.0"
 authors = ["The Servo Project Developers"]
+documentation = "https://docs.rs/stylo_atoms/"
+description = "Interned string type for the servo and stylo projects"
```

Not a native speaker, but "internal" sounds less archaic?

```suggestion
description = "Internal string type for the servo and stylo projects"
```

---

That would change the meaning here I think. The idea is to refer to the concept of [string interning](https://en.wikipedia.org/wiki/String_interning) rather than indicate that this library is internal to Servo.